### ISSUE TITLE
Doesn't work on ARM achitecture

### ISSUE THREAD

**Issue by jgb-solutions** [coollabsio/coolify#348]
trying to run coolify on one of the free ARM compute VM provided by Oracle Cloud. here's what I got 
```
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
```

Got the same trying to run other images not targeting ARM architecture.

**Comment by andrasbacsai** [comment]
Yeah, it looks like `bitnami/redis` does not support ARM. I will try to come up with a fix.

**Comment by andrasbacsai** [comment]
I find a quick fix. I'm releasing the new version soon (~20 mins)

**Comment by jgb-solutions** [comment]
cool!

**Comment by andrasbacsai** [comment]
You probably need to reinstall Coolify with `wget -q https://get.coollabs.io/coolify/install.sh -O install.sh; sudo bash ./install.sh -f`

**Comment by jgb-solutions** [comment]
I'll try it again and let you know.

**Comment by jgb-solutions** [comment]
```
docker logs -f coolify-redis
1:C 14 Apr 2022 22:19:09.896 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
1:C 14 Apr 2022 22:19:09.896 # Redis version=6.2.6, bits=64, commit=00000000, modified=0, pid=1, just started
1:C 14 Apr 2022 22:19:09.896 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
1:M 14 Apr 2022 22:19:09.897 * monotonic clock: POSIX clock_gettime
1:M 14 Apr 2022 22:19:09.898 * Running mode=standalone, port=6379.
1:M 14 Apr 2022 22:19:09.898 # Server initialized
1:M 14 Apr 2022 22:19:09.898 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
1:M 14 Apr 2022 22:19:09.905 * Ready to accept connections
1:M 14 Apr 2022 22:24:10.012 * 100 changes in 300 seconds. Saving...
1:M 14 Apr 2022 22:24:10.012 * Background saving started by pid 15
15:C 14 Apr 2022 22:24:10.020 * DB saved on disk
15:C 14 Apr 2022 22:24:10.020 * RDB: 0 MB of memory used by copy-on-write
1:M 14 Apr 2022 22:24:10.114 * Background saving terminated with success
```

```
docker logs -f coolify

> coolify@2.4.9 start /app
> npx prisma migrate deploy && npx prisma generate && npx prisma db seed && node build/index.js

Prisma schema loaded from prisma/schema.prisma
Datasource "db": SQLite database "prod.db" at "file:../db/prod.db"

16 migrations found in prisma/migrations


No pending migrations to apply.
npm notice
npm notice New minor version of npm available! 8.5.0 -> 8.7.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.7.0>
npm notice Run `npm install -g npm@8.7.0` to update!
npm notice
Prisma schema loaded from prisma/schema.prisma
Warning: Your current platform `linux-arm64-openssl-1.1.x` is not included in your generator's `binaryTargets` configuration ["linux-musl"].
To fix it, use this generator config in your schema.prisma:
generator client {
  provider      = "prisma-client-js"
  binaryTargets = ["native", "linux-musl"]
}
Note, that by providing `native`, Prisma Client automatically resolves `linux-arm64-openssl-1.1.x`.
Read more about deploying Prisma Client: https://github.com/prisma/prisma/blob/main/docs/core/generators/prisma-client-js.md


✔ Generated Prisma Client (3.11.1 | binary) to ./node_modules/.pnpm/@prisma+client@3.11.1_prisma@3.11.1/node_modules/@prisma/client in 595ms
You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client
```
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
```
┌─────────────────────────────────────────────────────────┐
│  Update available 3.11.1 -> 3.12.0                      │
│  Run the following to update                            │
│    npm i --save-dev prisma@latest                       │
│    npm i @prisma/client@latest                          │
└─────────────────────────────────────────────────────────┘
Running seed command `node prisma/seed.cjs` ...

🌱  The seed command has been executed.
Sentry Logger [log]: Integration installed: InboundFilters
Sentry Logger [log]: Integration installed: FunctionToString
Sentry Logger [log]: Integration installed: ContextLines
Sentry Logger [log]: Integration installed: Console
Sentry Logger [log]: Integration installed: Http
Sentry Logger [log]: Integration installed: OnUncaughtException
Sentry Logger [log]: Integration installed: OnUnhandledRejection
Sentry Logger [log]: Integration installed: LinkedErrors
Listening on 0.0.0.0:3000
```

still not working.

**Comment by andrasbacsai** [comment]
The prisma thing is a warning. Other than that, it looks okay. Is it not reachable in your browser?

**Comment by jgb-solutions** [comment]
keeps loading and loading. but not displaying anything.

**Comment by jgb-solutions** [comment]
just tried coolify on a new Vultr VM and it works fine. I guess it's the ARM thing that's the issue. Hopefully it's not the Oracle free VM that are the culprits.

**Comment by jgb-solutions** [comment]
I even tried one their Intel one and it doesn't work. Maybe that's something from their side.

**Comment by jgb-solutions** [comment]
Quick update: AMD and Intel work from Oracle. I had to open the port 3000 to talk to the server. I will try again on ARM and update this thread.

**Comment by dickermoshe** [comment]
@jgb-solutions 
Did you get the ARM servers to work ?
I have a similar issue where the `XX.XX.XX.XX:3000` page just hangs.
Opening port `3000/tcp` and `3000/udp` did not help.
Did you do any `iptables` changes within the VM ?

**Comment by jgb-solutions** [comment]
Hey @dickermoshe yes I got it to work but pulling the latest version at the time. Though it doesn't work as expected. I think last month or so I tried it again on an ARM machine. Everything seemed to work, I could deploy an Elixir app with Docker. But no Node apps wanted to run. They didn't use Docker. Just plain Next.js apps. Other than that No DB or analytics did work. They start then stop by themselves in a few seconds. I didn't bother opening another issue since Coolify is not something I'm planning to use in production (Not in the near future anyway) but just to play around on free Oracle servers.
